.gitignore

```diff
@@ -30,3 +30,5 @@ newrelic_agent.log
 backstop-tests/backstop_data/html_report/
 backstop-tests/backstop_data/bitmaps_test/
 backstop-tests/.idea
+backstop-tests/package.json
+backstop-tests/package-lock.json
```

Comment:
I think package-lock.json should be committed, I don't know about package.json though. https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
There isn't any need to do this as the package is installed as part of `./scripts/test-backstop`.